### DISCUSSION TITLE
Cannot access Jellyseerr over VPN (Tailscale)

### DISCUSSION THREAD

**Discussion by adambibor** [seerr-team/seerr#1118] [Q&A] [answered]
Hi, I'm fairly new to containerization, and I'm guessing that might be the source of the problem.
I run Jellyseerr on Windows and Podman, and use Tailscale to remotely access various services (Jellyfin, Sonarr, etc.). 
I can access all other services from my mobile or laptop without any problems, but not Jellyseerr, it only works locally. I have enabled the port 5055 in the Windows firewall to see if that's the problem, but that didn't help either. Do you have a solution for this situation?

**Comment by fallenbagel** [comment, marked as answer]
I would say hop onto discord as this is a support question and we provide support there.

Without knowing too much of how your setup is, from the surface it sounds to me that it's not bound to the tailscale interface.

Which is why im suggestion hopping onto discord as there are more people with your particular setup who can help you out + we carry out support there
